python-package/lightgbm/sklearn.py

```diff
@@ -76,7 +76,7 @@ def __call__(self, preds, dataset):
         elif argc == 3:
             grad, hess = self.func(labels, preds, dataset.get_group())
         else:
-            raise TypeError("Self-defined objective function should have 2 or 3 arguments, got %d" % argc)
+            raise TypeError(f"Self-defined objective function should have 2 or 3 arguments, got {argc}")
```

Comment:
Can you please remove these unrelated changes from this PR? I think they were meant for https://github.com/microsoft/LightGBM/pull/4181.

On whichever branch you use (based on my recommendation in https://github.com/microsoft/LightGBM/pull/4181), you can remove them with a command like `git reset --soft HEAD~1` followed by a force push.

Let me know if you get stuck!